### PR TITLE
GS Assets page

### PR DESCRIPTION
**Pull request by victorvalle**

### CODE REVIEW

File: src/markdown/blueprints/resources/gs-assets.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 'GS Assets'
```
Comment (plhnk) [inline src/markdown/blueprints/resources/gs-assets.mdx:2]: @rachelcyuan  @ryconn888 @abelhancock @jonmwood thoughts on this page name? is it clear enough?

**Review by plhnk — APPROVED** [review]
thanks @victorvalle!

File: src/markdown/blueprints/resources/gs-assets.mdx (hunk 1 above)
Comment (plhnk) [reply src/markdown/blueprints/resources/gs-assets.mdx:2]: i'm merging this — but please, put your thoughts, we can always adjust later :)

**Comment by ryconn888** [comment]
gs-assets seems pretty clear

**Pull request merged**